packages/customWidgets/calendar-web/src/components/Calendar.ts

```diff
@@ -199,6 +201,12 @@ class Calendar extends Component<CalendarProps, State> {
 
     private eventColor = (events: CalendarEvent) => ({ style: { backgroundColor: events.color } });
 
+    private dayGetter = (date: Date) => {
+        return this.props.disabledTill && this.props.disabledTill !== new Date("") && this.props.disabledTill > date
+            ? { className: "rbc-disabled abc", style: {} }
```

Comment:
class abc?

Reply:
It was a copy/paste from Freeks solution. I removed the 'abc' class